en/sensor/px4flow.md

```diff
@@ -0,0 +1,456 @@
+
+![PX4Flow v1.0](../../assets/hardware/sensors/px4flow/px4flow_v1.0_top_generated.png)
+
+Unlike many mouse sensors, it also works indoors and in low outdoor light conditions without the need for an illumination LED. It can be freely reprogrammed to do any other basic, efficient low-level computer vision task.
+
+* 168 MHz Cortex M4F CPU (128 + 64 KB RAM)
+* 752x480 MT9V034 image sensor, L3GD20 3D Gyro
+* 16 mm M12 lens (IR block filter)
+* Size 45.5 mm x 35mm
+* Power consumption 115mA / 5V
+
+{% youtube %}
+https://youtu.be/0Jpq6DU_HVg
+{% endyoutube %}
+
+
+## Where to Buy
+
+Order this module from:
+
+* [Unmanned Tech](http://www.unmannedtechshop.co.uk/px4flow-smart-camera-optical-flow-sensor/) (UK)
+* [UAV store](http://www.uav-store.de/sensoren/px4-flow-kit/) (Germany & EU)
+
+If out of stock the software-compatible, but not connector-compatible version can be used:
+
+* [PX4FLOW with Molex connectors](http://www.hobbyking.com/hobbyking/store/__66308__HK_Pilot32_Optical_Flow_Kit_With_Sonar.html)
+
+## Specifications
+
+* MT9V034 machine vision CMOS sensor with global shutter
+* Optical flow processing at 4x4 binned image at **400 Hz**
+* Superior light sensitivity with 24x24 μm super-pixels
+* Onboard 16bit gyroscope up to 2000°/s and 780 Hz update rate, default high precision-mode at 500°/s
+* Onboard sonar input and mount for [Maxbotix sonar sensors](../sensor/rangefinders.md#maxbotix-i2cxl-maxsonar-ez). (HRLV-EZ4 recommended, [SparkFun Product Link](https://www.sparkfun.com/products/11309))
+* USB bootloader
+* USB serial up to 921600 baud (including live camera view with [QGroundControl](http://qgroundcontrol.com/))
+* USB power option
+* **Does fit the MatrixVision Bluefox MV mounting holes (camera center off-centered)**
+
+![PX4Flow Top](../../assets/hardware/sensors/px4flow/px4flow_top.jpg) ![px4flow-bottom](../../assets/hardware/sensors/px4flow/px4flow_bottom.jpg) 
+
+
+# Pixhawk Setup
+
+> **Note** Applies to all PX4 FMU versions, including all versions of Pixhawk.
+
+- Update the firmware on PX4Flow using *QGroundControl* (in the top left menu, click on CONFIG, then on Firmware Upgrade)
+- Connect PX4Flow I2C to the Pixhawk I2C
+
+The module will be detected on boot, flow data should be coming through at 10Hz if the autopilot is connected via **USB**. 
+Flow data is transmitted via wireless at a lower rate.
+
+## Orientation
+
+The default orientation (meaning: zero rotation) is defined as Y on flow board pointing towards **front of vehicle** as shown in the following picture.
+
+![PX4Flow align with Pixhawk](../../assets/hardware/sensors/px4flow/px4flowalignwithpixhawk.jpg)
+
+
+The orientation of PX4Flow can be set using the parameter [MAV_SENSOR_ORIENTATION](https://mavlink.io/en/messages/common.html#MAV_SENSOR_ORIENTATION)
+
+> **Warning** This parameter can only be applied to PX4 flight stack and is **not** part of the sensor itself settings
+
+<!-- Remove below if orientation  param above is correct -->
```

Comment:
I commented this section because the SENS_FLOW_ROT no longer exists. I imagine it has been replaced by MAV_SENSOR_ORIENTATION but can't see where that happens. Can you confirm what is correct orientation param here?

Reply:
The parameter still exists: https://github.com/PX4/Firmware/blob/master/src/modules/sensors/sensor_params_flow.c#L54

MAV_SENSOR_ORIENTATION is for the distance sensor, used by the distance message: https://mavlink.io/en/messages/common.html#DISTANCE_SENSOR.
Since the device works via I2C, we don't need to mention MAV_SENSOR_ORIENTATION here.

Reply:
Thanks! Actually I was being mad - that's a message, not a parameter. I have fixed: https://github.com/PX4/px4_user_guide/pull/286/commits/15392f80e79fbfbf4dbe7cd749240627cfcc9b13